src/components/calculator.tsx

```diff
@@ -238,30 +250,11 @@ export default function Calculator() {
         </button>
         {calculatorMode === CalculatorMode[1] && (
           <>
-            <button
-              onClick={handleSin}
-              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
-            >
-              sin
-            </button>
-            <button
-              onClick={handleCos}
-              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
-            >
-              cos
-            </button>
-            <button
-              onClick={handleTan}
-              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
-            >
-              tan
-            </button>
-            <button
-              onClick={handleLogToE}
-              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
-            >
-              arcsin
-            </button>
+            {trigs.map((trig) => (
+              <CalcButton key={trig} onClick={handleTrig}>
```

Comment:
```
<CalcButton key={trig} onClick={() => handleTrig(trig)}>
```
You probably forgot to pass the argument :)

Reply:
The function is being handled in the `CalcButton` component

Reply:
What I am passing is the name of the function. `CalcButton` handles the calling of the function.